mealy/error_analyzer.py

```diff
@@ -139,7 +139,8 @@ def leaf_ids(self):
             self._compute_leaf_ids()
         return self._leaf_ids
 
-    def fit(self, x, y, max_nr_rows=ErrorAnalyzerConstants.MAX_NUM_ROW):
+    def fit(self, x, y, max_nr_rows=ErrorAnalyzerConstants.MAX_NUM_ROW,
```

Comment:
I propose that we put all the parameters like `max_nr_rows` and `tree_params` in the `__init__` part of the object, as others scikit model